### PR TITLE
Update ClusterRegistration type

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by tbrisker** [comment]
@pvasant PTAL, you added these fields a couple of years ago :)

File: model/clusters_mgmt/v1/cluster_registration_type.model [hunk 1]
```diff
@@ -15,30 +15,38 @@ limitations under the License.
...
-	//
-	// ```json
-	// {
-	//   "external_id": "d656aecf-11a6-4782-ad86-8f72638449ba"
-	// }
-	// ```
 	ExternalID String
+
+	// Optional Display Name for the cluster.
+	DisplayName String
```
Comment (ciaranRoche) [inline model/clusters_mgmt/v1/cluster_registration_type.model:48]: Just curious, what is the status with DisplayName, I was under the impression we had removed DisplayName from CS or has that change been reverted?

Comment (tbrisker) [reply model/clusters_mgmt/v1/cluster_registration_type.model:48]: it was reverted

Comment (ciaranRoche) [reply model/clusters_mgmt/v1/cluster_registration_type.model:48]: Ah cool cool cool

Comment (pvasant) [reply model/clusters_mgmt/v1/cluster_registration_type.model:48]: @tbrisker @ciaranRoche  we dont want to use display name it was reverted in CS until AMS fixes their logic. however we want new users to stop using the display name . can you please remove it

**Pull request merged**